packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1284,6 +1284,9 @@ const schema: SchemaType<DbPost> = {
     editableBy: [userOwns, 'admins'],
     insertableBy: ['members'],
     control: "FMCrosspostControl",
+    tooltip: fmCrosspostBaseUrlSetting.get() && ["https://forum.effectivealtruism.org", "https://forum.effectivealtruism.org/"].includes(fmCrosspostBaseUrlSetting.get()!) ?
```

Comment:
I don't want to play games with trailing slashes.

Reply:
```suggestion
    tooltip: fmCrosspostBaseUrlSetting.get()?.includes("forum.effectivealtruism.org") ?
```
Maybe?

Reply:
(or a regex)